doc/nrf/templates/sample_README.rst

```diff
@@ -1,57 +1,81 @@
+   * Mention additional requirements after the table.
+   * If SPM is included in the sample, add ``.. include:: /includes/spm.txt`` to include the standard text that states this.
+
+The sample supports the following development kits:
+
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set1_start
+   :end-before: set1_end
+
+The sample also requires ...
+
+.. include:: /includes/spm.txt
 
 
 Overview
 ********
 
-You can use this sample application as a starting point to implement a disco light.
+You can use this sample as a starting point to implement a disco light.
 
-The sample applicaton uses the ``:ref:RST link`` library to control the LEDs. In addition, it uses the ``:ref:RST link`` sound sensor and hooks up to some internet service to download cool blinking sequences.
+The sample uses the ``:ref:RST link`` library to control the LEDs.
+In addition, it uses the ``:ref:RST link`` sound sensor and hooks up to some Internet service to download cool blinking sequences.
 
 .. tip::
-   Continue the explanation on what this example is about.
+   Continue the explanation on what this sample is about.
    What does it show, and why should users try it out?
-   What is the practical use? How can users extend this example? What libraries are used?
-   When linking, link to the conceptual documentation of other modules - these will in turn link to or contain the API documentation.
+   What is the practical use?
+   How can users extend this sample?
+   What libraries are used (link to them)?
+
 
 Some title*
 ===========
 .. note::
-   Add subsections for technical details. Give them a suitable title (sentence style capitalization, thus only the first word capitalized).
+   Add subsections for technical details.
+   Give them a suitable title (sentence style capitalization, thus only the first word capitalized).
+   If there is nothing important to point out, do not include any subsections.
 
-The sample application repeatedly calls function ABC, which ...
-
-.. tip::
-   Do not state what people can see and understand from looking at the code. Instead, clarify general concepts or explain parts of the implementation that might be unintuitive for some reason. If there's nothing important to point out, do not include any subsections.
-
-
-Requirements
-************
-
-* One of the following development boards:
-
-  * nRF52840 Development Kit board (PCA10056)
-  * nRF52 Development Kit board (PCA10040)
-  * nRF51 Development Kit board (PCA10028)
 
-* A Shazam account
-* A disco ball
+The sample application repeatedly calls function ABC, which ...
 
 .. tip::
-   Unless the sample is meant to go upstream, require a Nordic board, not "any board with BLE". Be specific - "any Nordic board" will also mean nRF24 ... When listing the supported boards, give both the name (nRF52 Development Kit board) and the board number (PCA10040).
+   Do not state what people can see and understand from looking at the code.
```

Comment:
This is a bit of a risky statement. Developers who are familiar with their code will always assume that people can understand more from just looking at the code than they actually can. I think putting it like this might lead a dev to not describe something that would clarify how the sample works because they will assume it's obvious.

Reply:
Clarified.